Add the Console UI tab for the tutorial on triggering events (#7944)
kyma-project/kyma: docs/serverless-v2/08-04-trigger-lambda-with-event.md

      broker: default
      filter:
        attributes:

>           eventTypeVersion: $EVENT_VERSION

Reviewer: ```suggestion
          eventtypeversion: $EVENT_VERSION
```

This attribute is not camel-cased

    EOF
    ```

    </details>
    <details>
    <summary label="console-ui">
    Console UI
    </summary>

> **NOTE:** Serverless v2 is an experimental feature, and it is not enabled by default in the Console UI. To use its **Functions [preview]** view, enable **Experimental functionalities** in the **General Settings** view before you follow the steps. Refresh the page after enabling this option.

1. From the drop-down list in the top navigation panel, select the Namespace in which your Application exposes events.

2. Go to the **Functions [preview]** view at the bottom of the left navigation panel and navigate to your lambda.

3. Once in the lambda view, select **Add Event Trigger** in the **Event Triggers** section.

> 4. Select the event type and version that you want to be a trigger for your lambda and select **Add** to confirm changes.

Reviewer 2: ```suggestion
4. Select the event type and version that you want to use as a trigger for your lambda and select **Add** to confirm changes.
```

The `Event Triggers are successfully created` message appears and you will see the trigger available in the **Event Triggers** section in your lambda.

    </details>
</div>

## Test the trigger

> > **CAUTION:** Before you follow steps in this section and send an example event, bear in mind that your changes will be propagated to all services subscribed to the event type you send.

Reviewer: ```suggestion
> **CAUTION:** Before you follow steps in this section and send an example event, bear in mind that your events will be propagated to all services subscribed to the event type you send.
```

Author: `> **CAUTION:** Before you follow steps in this section and send a sample event, bear in mind that it will be propagated to all services subscribed to this event type.`
